styles.js

```diff
@@ -63,4 +70,58 @@ export default createGlobalStyle`
     --padding-bg-md: 20px;
     --padding-large: 24px ;
     --padding-extra-large: 32px ;
-  }`;
+  }
+${(props) =>
+  props.isDarkMode &&
+  css`
+    :root {
+      --color-primary: #74c69d;
+      --color-primary-light: #a8e6cf;
+      --color-primary-dark: #3b7a57;
+
+      --color-secondary: #292929ff;
+      --color-secondary-dark: #1f1f1fff;
+      --color-secondary-rgba: rgba(245, 241, 230, 0.8);
+
+      --color-white: #ffffff;
+      --color-black: #000000;
+
+      --color-neutral-dark: #e0e0e0;
+      --color-neutral-medium: #a0a0a0;
+      --color-neutral-light: #74c69d;
+
+      --color-text-white: #ffffff;
+      --color-text-dark: #f5f5f5;
+      --color-text-medium: #bdbdbd;
+
+      --color-light-grey: #444;
+
+      --color-accent: #ffd166;
+      --color-alert: #ff6b6b;
+
+      --radius-lg: 25px;
+      --radius-bg-md: 15px;
+      --radius-md: 10px;
+      --radius-sm: 5px;
+
+      --border-sm-dark: 1px solid #555;
+
+      --font-primary: "Playfair Display";
+      --font-secondary: "Nunito";
+
+      --font-size-xl: 24px;
+      --font-size-lg: 20px;
+      --font-size-md: 16px;
+      --font-size-sm: 14px;
+
+      --box-shadow-md: 0px 2px 10px rgba(0, 0, 0, 0.7);
+
+      --padding-small: 8px;
+      --padding-bg-sm: 10px;
+      --padding-medium: 16px;
+      --padding-bg-md: 20px;
+      --padding-large: 24px;
+      --padding-extra-large: 32px;
```

Comment:
your naming pattern does not make sense in dark mode. neutral dark is a very light color, neutral-light is dark, text-white is still while and text-dark is also light. Please update your variable names to not describe how the color looks but rather where it is used: color-background, color-foreground, color-background-elevated etc. Please do this as a team so that everyone is on the same page about the naming convention.